docs: cover Vega ES|QL panels in Fast mode docs (#7428) (#7485)
elastic/docs-content: explore-analyze/visualize/custom-visualizations-with-vega.md

}
```

> {applies_to}`stack: preview 9.5` {applies_to}`serverless: preview` If a **Vega** or **Vega-Lite** panel uses an {{esql}} data source, it becomes subject to the {icon}`bolt` [**Fast mode**](../query-filter/languages/esql-kibana.md#approximation-fast-mode) option on its dashboard, which returns faster, estimated results for `STATS` aggregations.

Reviewer: What do you think about:

```suggestion
{applies_to}`stack: preview 9.5` {applies_to}`serverless: preview` If a **Vega** or **Vega-Lite** panel uses an {{esql}} data source, you can use the {icon}`bolt` [**Fast mode**](../query-filter/languages/esql-kibana.md#approximation-fast-mode) option on its dashboard to return faster, estimated results for `STATS` aggregations.
```

Author: I think this changes a little bit the message but I think I see what you mean
```suggestion
{applies_to}`stack: preview 9.5` {applies_to}`serverless: preview` **Vega** and **Vega-Lite** panels that use an {{esql}} data source are subject to the {icon}`bolt` [**Fast mode**](../query-filter/languages/esql-kibana.md#approximation-fast-mode) dashboard option. When this option is active on a dashboard, these panels return faster, estimated results for `STATS` aggregations.
```

Reviewer: Ah, I understand now, I think I lacked the context. This works!
